test/validation/api/event/event.c

```diff
@@ -33,6 +33,7 @@ static void event_test_free(void)
 
 	for (i = 0; i < EVENT_BURST; i++) {
 		buf = odp_buffer_alloc(pool);
+		CU_ASSERT(odp_event_is_valid(odp_buffer_to_event(buf)) == 1);
```

Comment:
Similar checks could be added for `ODP_EVENT_CRYPTO_COMPL` and `ODP_EVENT_IPSEC_STATUS`events.

Reply:
done in v3